vault/resource_auth_backend.go

```diff
@@ -44,7 +44,6 @@ func AuthBackendResource() *schema.Resource {
 
 			"description": {
 				Type:        schema.TypeString,
-				ForceNew:    true,
```

Comment:
I think this might not be enough because the code that runs the tune (https://github.com/hashicorp/terraform-provider-vault/blob/f8dbbc626c942b0f66d23e8dc9ade1cefdb80aa9/vault/auth_mount.go#L13-L27) doesn't include `description` as far as I can tell.

There's also some other parameters that shouldn't force new as well:

https://github.com/hashicorp/terraform-provider-vault/blob/f8dbbc626c942b0f66d23e8dc9ade1cefdb80aa9/vault/resource_auth_backend.go#L57

https://github.com/hashicorp/terraform-provider-vault/blob/f8dbbc626c942b0f66d23e8dc9ade1cefdb80aa9/vault/resource_auth_backend.go#L68

If you're able to update these, that would be great! If not I can take over the PR and get it fixed.

Thanks!

Reply:
Hey @jasonodonnell , I'm totally happy for you to take this over :). As I found above as well, it seems that this affects a few different resources, so is probably best handled more holistically as well anyway.